scripts/variantstore/wdl/extract/populate_alt_allele_table.py

```diff
@@ -33,8 +33,8 @@ def populate_alt_allele_table(query_project, vet_table_name, fq_dataset, sa_key_
     fq_vet_table = f"{fq_dataset}.{vet_table_name}"
     query_with = f"""INSERT INTO `{fq_dataset}.alt_allele`
                 WITH 
-                  position1 as (select * from `{fq_vet_table}` WHERE call_GT IN ('0/1', '1/0', '1/1', '0|1', '1|0', '1|1', '0/2', '0|2','2/0', '2|0')), 
-                  position2 as (select * from `{fq_vet_table}` WHERE call_GT IN ('1/2', '1|2', '2/1', '2|1'))"""
+                  position1 as (select * from `{fq_vet_table}` WHERE call_GT IN ('0/1', '1/0', '1/1', '0|1', '1|0', '1|1', '0/2', '0|2','2/0', '2|0') AND sample_id NOT IN (SELECT sample_id from {fq_dataset}.sample_info WHERE withdrawn IS NOT NULL)),
+                  position2 as (select * from `{fq_vet_table}` WHERE call_GT IN ('1/2', '1|2', '2/1', '2|1') AND sample_id NOT IN (SELECT sample_id from {fq_dataset}.sample_info WHERE withdrawn IS NOT NULL))"""
```

Comment:
Not sure if this is better but it does use the partition key on the join, also presumes `fq_sample_info` is defined and set appropriately

```suggestion
                  position1 as (select vet.* from `{fq_vet_table}` vet join `{fq_sample_info}` info on vet.sample_id = info.sample_id WHERE info.withdrawn is null and vet.call_GT IN ('0/1', '1/0', '1/1', '0|1', '1|0', '1|1', '0/2', '0|2','2/0', '2|0')),
                  position2 as (select vet.* from `{fq_vet_table}` vet join `{fq_sample_info}` info on vet.sample_id = info.sample_id WHERE info.withdrawn is null and vet.call_GT IN ('1/2', '1|2', '2/1', '2|1'))"""
```